src/cmd/connect.go

```diff
@@ -47,7 +47,7 @@ var (
 		Aliases: []string{"l"},
 		Short:   "List all available connection shortcuts.",
 		Run: func(cmd *cobra.Command, args []string) {
-			k8s.PrintConnectTable()
+			cluster.NewClusterOrDie().PrintConnectTable()
```

Comment:
Thoughts on changing the NewCluster* functions to NewClusterClient*?  They are creating a new struct that represents the cluster, but not actually creating a new k8s cluster

Reply:
that might make sense, I'd be happy to discuss that and `NewPackage()` as well to decide on a common pattern.